portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,93 @@
+/*Handle background-change request */
+async function getRandomImageAsync() {
+    let response = await fetch("/background-change");
+    console.log("Got respone");
+    let imageURL = await response.text();
+    console.log(imageURL);
+    document.getElementById("bg-image").style.background = "linear-gradient(rgba(0,0,0,0.5),rgba(0,0,0,0.5)), url(" + imageURL +") no-repeat fixed center";
+};
+
+function isLetter(s) {
+  let alphaExp = /^[a-zA-Z() ]+$/;
+  return alphaExp.test(s);
+} 
+
+async function validateForm() {
+  let name = document.getElementById("name").value;
+  let content = document.getElementById("content").value;
+  if (!isLetter(name)) {
+    alert("JS: Name must alphabet letter only.");
```

Comment:
After a bit of research, I thought that the front end validation was necessary because otherwise invalid data would be sent to the server and exceptions would be throw out all the times. On the other hand, I also thought back end validation was necessary because otherwise, evil people would be able to get around the front end validation and send hazardous data to the server. On this post (https://www.quora.com/Should-I-sanitize-inputs-clientside-or-serverside), I've seen similar arguments as well. So that was my thought, but it is easy to mess up if both sides aren't in sync like you said. What is your thought?